smart-contracts/contracts/cl-vault/src/vault/deposit.rs

```diff
@@ -84,24 +318,84 @@ pub(crate) fn execute_exact_deposit(
         mint_to_address: env.clone().contract.address.to_string(),
     };
 
-    let mut resp = Response::new()
-        .add_attribute("method", "execute")
-        .add_attribute("action", "exact_deposit")
-        .add_attribute("amount0", deposit.0)
-        .add_attribute("amount1", deposit.1)
-        .add_message(mint_msg)
-        .add_attribute("mint_shares_amount", user_shares)
-        .add_attribute("receiver", recipient.as_str());
-
-    if let Some((bank_msg, bank_attr)) = refund_bank_msg(
-        recipient,
-        Some(coin(refund.0.u128(), pool.token0)),
-        Some(coin(refund.1.u128(), pool.token1)),
-    )? {
-        resp = resp.add_message(bank_msg).add_attributes(bank_attr);
+    Ok((mint_msg, user_shares))
+}
+
+#[allow(clippy::too_many_arguments)]
+fn calculate_swap_amount(
+    deps: DepsMut,
+    env: &Env,
+    pool_config: PoolConfig,
+    swap_direction: SwapDirection,
+    token_in_amount: Uint128,
+    swappable_amount: (Uint128, Uint128),
+    deposit_amount_in_ratio: (Uint128, Uint128),
+    max_slippage: Decimal,
+    recipient: &Addr,
+) -> Result<SwapCalculationResult, ContractError> {
+    // TODO check that this math is right with spot price (numerators, denominators) if taken by legacy gamm module instead of poolmanager
+    // TODO check on the twap_window_seconds (taking hardcoded value for now)
```

Comment:
Is this todo for this PR?

Reply:
it was added earlier during debugging accounting error, removed.